src/pts/pyspark/ontoma_lut_generation.py

```diff
@@ -0,0 +1,53 @@
+"""Generate disease lookup tables with OnToma."""
+
+from loguru import logger
+from ontoma import OnToma
+from ontoma import DiseaseCuration, OpenTargetsDisease
+
+from pts.pyspark.common.session import Session
+
+
+def ontoma_lut_generation(
+    source: dict[str, str],
+    destination: dict[str, str],
+    properties: dict[str, str],
+) -> None:
+    spark = Session(app_name='ontoma', properties=properties)
+
+    logger.info(f'load data from: {source}')
+    disease_index = spark.load_data(path=source['disease_index'], format='parquet')
```

Comment:
Parquet is the default format
```suggestion
    disease_index = spark.load_data(path=source['disease_index'])
```